external/downstream/pasture/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
-if(ENABLE_psi4pasture)
-  ExternalProject_Add(psi4pasture
-    DEPENDS psi4_core
+if(ENABLE_pasture)
+  ExternalProject_Add(pasture
+    DEPENDS psi4-core
     GIT_REPOSITORY https://github.com/psi4/psi4pasture
     CMAKE_ARGS -DENABLE_ccsort=${ENABLE_ccsort}
                -DENABLE_transqt2=${ENABLE_transqt2}
```

Comment:
This still seems to be an issue, I take back my first comment though you are 100% right using an external pybind11 would make that argument invalid. 

I think the best way to go would to be passing `-Dpybind11_DIR=${pybind11_DIR}` to the external project. If an external pybind is used that variable is how it would be communicated to the build, and if we are building pybind internally that variable is set after the external project call for [pybind](https://github.com/psi4/psi4/pull/554/files?diff=unified#diff-5e8400e93d83200f7e000603ba1d3cc7R20)